### ISSUE TITLE
Greener N' Cleaner Cooldown doesnt decrease with Mork's Waaagh

### ISSUE THREAD

**Issue by Sixmoons48** [WarEmu/WarBugs#16514]
**Expected behavior and actual behavior:**
Expected behavior, when Mork's Waaagh is built by dealing damage, decrease cooldown of Greener N' Cleaner. Actual behavior is that the cooldown of 5 seconds stays 5 seconds even after building 5 stacks of Mork's waaagh

**Steps to reproduce the problem:**
Get an ailment or curse on you while having morks waaagh and use cleanse to remove one, Cooldown wont change

**Testing Screenshots/Videos/Evidences (always needed):**

Can show screenshots if really need, have some but hard to show exactly

**Comment by Kaelang** [comment]
Appreciate screenshots might be difficult to provide in this circumstance, however are you able to screen record and provide a video?

Thanks.

**Comment by Sixmoons48** [comment]
Trying to find a way to get a vid into here for you

**Comment by Sixmoons48** [comment]
![1 Stack of Mork](https://user-images.githubusercontent.com/21229715/80311426-35c9e980-87a5-11ea-9124-31d9f8742221.gif)
![5 Stacks of Mork](https://user-images.githubusercontent.com/21229715/80311432-3bbfca80-87a5-11ea-8816-719bc0bac2f0.gif)
![1 Stack of Gork](https://user-images.githubusercontent.com/21229715/80311470-67db4b80-87a5-11ea-9e72-dd3b09395d20.gif)

**Comment by Sixmoons48** [comment]
ability is on bottom right of right hot bars, cleanse, always 5 seconds ^^

**Comment by Kaelang** [comment]
This is really difficult to see - is there anyway you could upload it to a service like YouTube? Appreciate that you've tried making these gifs, but the quality has come out pretty low.

**Comment by Kaelang** [comment]
Deleted previous comment, now I know what I'm looking for - was able to see it. Thank you.

**Comment by grunbag** [comment]
Waaaagh reduxe cast time and not cool down.

**Comment by Sixmoons48** [comment]
thats wrong grunbad, cleanse is insta cast, so there is no cast time reduction involved

**Comment by Kaelang** [comment]
But he's saying that the skill reduces the cast time. So it doesn't effect cleanse.

**Comment by grunbag** [comment]
Right it seems special and tooltip says it should reduxe cooldown . Could you try on cleansing an ally instead of yourself ?

**Comment by Sixmoons48** [comment]
yes will test and send more evidence

**Comment by Sixmoons48** [comment]
![Hnet-image](https://user-images.githubusercontent.com/21229715/80339362-19b95d00-8824-11ea-929f-a2b7a9e230fa.gif)

with 5 stacks of mork, (the one that is supposed to lower cooldown). on allies it lowers CD by 2,5 seconds (50%), casting on self is no CD reduction, tooltip doesnt specify this should be case, having it be on self and allow you to cleanse quickly while playing dps shammy sounds like a fun build

**Comment by Sixmoons48** [comment]
dunno if i need to bump or if yall noticed my update

**Comment by Kaelang** [comment]
We got it - thanks.
